core/src/test/scala/integration/kafka/api/AuthorizerIntegrationTest.scala

```diff
@@ -433,7 +435,7 @@ class AuthorizerIntegrationTest extends KafkaServerTestHarness {
     this.consumers.head.commitSync(Map(tp -> new OffsetAndMetadata(5)).asJava)
   }
 
-  @Test(expected = classOf[TopicAuthorizationException])
+  @Test(expected = classOf[KafkaException])
```

Comment:
Let's see how KIP-4 will change things and we'll carry on the discussion in KAFKA-3727